mdata/notifierKafka/cfg.go

```diff
@@ -44,104 +38,42 @@ var messagesSize = stats.NewMeter32("cluster.notifier.kafka.message_size", false
 func init() {
 	fs := flag.NewFlagSet("kafka-cluster", flag.ExitOnError)
 	fs.BoolVar(&Enabled, "enabled", false, "")
-	fs.StringVar(&brokerStr, "brokers", "kafka:9092", "tcp address for kafka (may be given multiple times as comma separated list)")
-	fs.StringVar(&topic, "topic", "metricpersist", "kafka topic")
-	fs.StringVar(&partitionStr, "partitions", "*", "kafka partitions to consume. use '*' or a comma separated list of id's. This should match the partitions used for kafka-mdm-in")
-	fs.StringVar(&partitionScheme, "partition-scheme", "bySeries", "method used for partitioning metrics. This should match the settings of tsdb-gw. (byOrg|bySeries)")
-	fs.StringVar(&offsetStr, "offset", "last", "Set the offset to start consuming from. Can be one of newest, oldest,last or a time duration")
-	fs.StringVar(&dataDir, "data-dir", "", "Directory to store partition offsets index")
 	fs.DurationVar(&offsetCommitInterval, "offset-commit-interval", time.Second*5, "Interval at which offsets should be saved.")
+	fs.IntVar(&batchNumMessages, "batch-num-messages", 10000, "Maximum number of messages batched in one MessageSet")
```

Comment:
ass all of these flag vars are just going to be put into a kafka.ConsumerConf{} why just just a package global `consumerConfig`
eg
```
var consumerConfig kafka.ConsumerConf
func init() {
  consumerConfig = kafka.NewConfig()
  fs := flag.NewFlagSet("kafka-cluster", flag.ExitOnError)
  fs.IntVar(&consumerConfig.BatchNumMessages, "batch-num-messages", 10000, "Maximum number of messages batched in one MessageSet")
  ...
```

Reply:
good idea